pkg/authorization/authorizer/scope/converter.go

```diff
@@ -95,6 +96,17 @@ func (userEvaluator) Validate(scope string) error {
 	return fmt.Errorf("unrecognized scope: %v", scope)
 }
 
+func (userEvaluator) Describe(scope string) string {
+	switch scope {
+	case UserIndicator + UserInfo:
+		return "information about you, including username, identity names, and group membership"
```

Comment:
If this information is going to be returned to a user, should it be properly capitalized and punctuated?

Also, grammar-nit; needs a colon (but I agree on the use of the Oxford Comma):
"Information about you, including: username, identity names, and group membership."